### PR TITLE
Added watchtower

### LINKED ISSUE #10478: Request: Watchtower
### Brand Name

Watchtower

### Website

https://containrrr.dev/watchtower/

### Popularity Metric

The Similarweb Global rank is 839.752. See https://www.similarweb.com/es/website/containrrr.dev/?utm_source=addon&utm_medium=firefox&utm_content=footer&utm_campaign=cta-button

### Official Resources for Icon and Color

From the website logo: https://containrrr.dev/watchtower/images/logo-450px.png

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by nirvikpurkait**
image

**Issue:** closes #10478

**Similarweb rank:**
**839,752**

#### Consideration:

Github start: 16.3k
Repository is [still active](https://github.com/containrrr/watchtower/commits/main/)
[star history](https://star-history.com/#containrrr/watchtower&Date) is upward trajectory

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by nirvikpurkait** [comment]
Every test, lint passed, github action failed due to:

```
FetchError: request to https://github.com/editorconfig-checker/editorconfig-checker/releases/download/2.8.0/ec-linux-amd64.tar.gz failed, reason: connect ETIMEDOUT 140.82.112.3:443
```

Can we re run the checks?

**Comment by LitoMore** [comment]
Hi @nirvikpurkait, it should be a connection issue related to your local environment. As you can see the HTTP request timed out when connecting to a GitHub public URL.

You can use `npm run svglint` command to run SVG linters only to bypass the EditorConfig Checker.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -16341,6 +16341,11 @@
             "hex": "4946DD",
             "source": "https://github.com/wasmerio/wasmer.io/blob/0d425f5b4ace56496e75278e304f54492c46adde/public/images/icon.svg"
         },
+        {
+            "title": "Watchtower",
+            "hex": "406170",
+            "source": "https://containrrr.dev/watchtower/"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:16347]:
```suggestion
            "source": "https://containrrr.dev/watchtower"
```

**Review by PeterShaggyNoble — APPROVED** [review]
As far as manual vectorisations go, this is nigh on perfect - very nice work, @nirvikpurkait 👍🏻

It _does_ line up a _little_ better if the height is resized to 24dp but I don't thing the differences are significant enough to block us from merging this as-is, but I'll ask one of the @simple-icons/pr-reviewers to throw another eye over it, too.

For the colour, I think we should go with `#2d383e`, which is the website's background colour - what do you reckon?

**Comment by service-paradis** [comment]
> As far as manual vectorisations go, this is nigh on perfect - very nice work, @nirvikpurkait 👍🏻
> 
> It _does_ line up a _little_ better if the height is resized to 24dp but I don't thing the differences are significant enough to block us from merging this as-is, but I'll ask one of the @simple-icons/pr-reviewers to throw another eye over it, too.
> 
> For the colour, I think we should go with `#2d383e`, which is the website's background colour - what do you reckon?

Looks good to me, good job @nirvikpurkait 🎉!

For the color I would opt for the header background color (`#416271`). It stays the same in light mode or dark mode and is called `--md-primary-fg-color` in their stylesheet.

**Review by service-paradis — APPROVED** [review]

File: _data/simple-icons.json [hunk 2]
```diff
@@ -16341,6 +16341,11 @@
             "hex": "4946DD",
             "source": "https://github.com/wasmerio/wasmer.io/blob/0d425f5b4ace56496e75278e304f54492c46adde/public/images/icon.svg"
         },
+        {
+            "title": "Watchtower",
+            "hex": "406170",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:16346]:
```suggestion
            "hex": "416271",
```

**Pull request merged**